utils/stubs.js

```diff
@@ -46,6 +46,33 @@ module.exports = {
     default: "",
     name: "new-selector"
   },
+  story_description: {
```

Comment:
let's talk about `stubs`

Are these just default values for columns of a new row in the database? Can we just set the default values directly in the database schema and not have this file at all?

Reply:
Yes, you are correct. The original purpose of this file was for making *tree elements*, which needed additional metadata (repeated, visual properties like `hasCaret`).  But I followed this down to database elements, and DEFAULT values in the schema would suffice.  I will make these changes.